source/atlas-search.txt

```diff
@@ -0,0 +1,127 @@
+categorizes data in a searchable format. To learn how to create an Atlas Search Index 
+see :ref:`pymongo-atlas-search-index`.
+
+Search Your Data
+----------------
+
+To use the ``$search`` aggregation pipeline stage, you must specify an Atlas Search query
+operator that indicates the type of query you want to run. You can also optionally specify
+a collector that groups results by values or ranges. To view a table of all the operators
+and collectors available with Atlas Search, see :atlas:`Use Operators and Collectors in 
+Atlas Search Queries </atlas-search/operators-and-collectors>`. 
+
+The following example uses the ``phrase`` operator, which performs a search for documents
+that contain an ordered sequence of terms. To learn more about the ``phrase`` operator, see
+:atlas:`Phrase </atlas-search/phrase>` in the Atlas guide. 
+
+The example performs a basic search of the ``title`` field for the query string ``"new york"``.
+The query also includes the following stages:
+
+- :pipeline:`$limit`, to limit the output to 10 results.
+- :pipeline:`$project`, to exclude all fields except
+  ``title`` and add a field named ``score``.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: python
+
+      client = pymongo.MongoClient("<connection-string>")
+      result = client["sample_mflix"]["movies"].aggregate([
+               {
+                  "$search": {
+                  "index": "pymongoindex",
+                  "phrase": {
+                  "path": "title",
+                  "query": "new york"
+                 }
+                }
+               },
+               { "$limit": 10 },
+               {
+                 "$project": {
+                 "_id": 0,
+                 "title": 1,
+                 "score": { "$meta": "searchScore" }
+                 }
+                }
+              ])
+
+      for i in result:
+         print(i)
+
+   .. output::
+      :language: none
+      :visible: false
+
+      { title: 'New York, New York', score: 6.786321640014648 }
+      { title: 'New York', score: 6.258549213409424 }
+      { title: 'New York Stories', score: 5.3813982009887695 }
+      { title: 'New York Minute', score: 5.3813982009887695 }
+      { title: 'Synecdoche, New York', score: 5.3813982009887695 }
+      { title: 'New York Doll', score: 5.3813982009887695 }
+      { title: 'Little New York', score: 5.3813982009887695 }
+      { title: 'Escape from New York', score: 4.719893455505371 }
+      { title: 'Naked in New York', score: 4.719893455505371 }
+      { title: 'Autumn in New York', score: 4.719893455505371 }
+
+Additional Information
+----------------------
+
+To learn more about the different Atlas Search :atlas:`operators 
+</atlas-search/operators-and-collectors>` and other queries you can run, see the
+Atlas Search :atlas:`documentation </atlas-search>`. 
+
+You can view more query examples througout the Atlas :atlas:`documentation 
+</atlas-search>`.
```

Comment:
S: It's not entirely clear where each of these links will take you until you click on them. Also this section links to the same page twice, so I think we can eliminate the first link to the general Atlas Search docs.
```suggestion
To learn more about the available Atlas Search operators, see the :atlas:`Operators and Collectors </atlas-search/operators-and-collectors>` guide in the MongoDB Atlas documentation.

For more information about Atlas Search, and to view more query examples, see the
:atlas:`Atlas Search documentation </atlas-search>`. 
```